pkg/cmd/experimental/syncgroups/config/types.go

```diff
@@ -0,0 +1,148 @@
+package config
```

Comment:
They fit better in ldaputil.  Be sure to keep them separate from the other pull/commit.